keyboards/royal_kludge/rk61_rgb/config_led.h

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2020 euwbah <euwbah@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+// Backlight configuration
+#define RGB_MATRIX_FRAMEBUFFER_EFFECTS
+#define RGB_MATRIX_KEYPRESSES
+#define RGB_MATRIX_MAXIMUM_BRIGHTNESS 255
+#define RGBLIGHT_HUE_STEP 10
+#define RGBLIGHT_LIMIT_VAL 255
+#define RGBLIGHT_SAT_STEP 17
+#define RGBLIGHT_VAL_STEP 17
```

Comment:
```suggestion
```
RGBLIGHT subsystem is not used, but RGBMATRIX. These will do nothing

If you desire that functionality, use the proper flags